pkg/xray/resource_xray_license_policy_test.go

```diff
@@ -0,0 +1,361 @@
+					resource.TestCheckResourceAttr(fqrn, "rules.0.actions.0.block_download.0.unscanned", "true"),
+					resource.TestCheckResourceAttr(fqrn, "rules.0.criteria.0.allowed_licenses.0", "Apache-1.0"),
+					resource.TestCheckResourceAttr(fqrn, "rules.0.criteria.0.allowed_licenses.1", "Apache-2.0"),
+					resource.TestCheckResourceAttr(fqrn, "rules.0.criteria.0.multi_license_permissive", "false"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccLicensePolicy_createBlockFalse(t *testing.T) {
+	policyName := "terraform-license-policy-6"
+	policyDesc := "policy created by xray acceptance tests"
+	ruleName := "test-license-rule-6"
+	gracePeriod := 5
+	resourceName := "policy-" + strconv.Itoa(randomInt())
+	fqrn := "xray_license_policy." + resourceName
+	multiLicense := "false"
+	blockUnscanned := "false"
+	blockActive := "false"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		CheckDestroy:      testAccCheckLicensePolicyDestroy(fqrn),
+		ProviderFactories: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccXrayLicensePolicy_createAllowedLic(resourceName, policyName, policyDesc,
+					ruleName, multiLicense, gracePeriod, blockUnscanned, blockActive),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(fqrn, "name", policyName),
+					resource.TestCheckResourceAttr(fqrn, "description", policyDesc),
+					resource.TestCheckResourceAttr(fqrn, "rules.0.name", ruleName),
+					resource.TestCheckResourceAttr(fqrn, "rules.0.actions.0.build_failure_grace_period_in_days", strconv.Itoa(gracePeriod)),
+					resource.TestCheckResourceAttr(fqrn, "rules.0.actions.0.custom_severity", "High"),
+					resource.TestCheckResourceAttr(fqrn, "rules.0.actions.0.fail_build", "true"),
+					resource.TestCheckResourceAttr(fqrn, "rules.0.actions.0.block_download.0.active", "false"),
+					resource.TestCheckResourceAttr(fqrn, "rules.0.actions.0.block_download.0.unscanned", "false"),
+					resource.TestCheckResourceAttr(fqrn, "rules.0.criteria.0.allowed_licenses.0", "Apache-1.0"),
+					resource.TestCheckResourceAttr(fqrn, "rules.0.criteria.0.allowed_licenses.1", "Apache-2.0"),
+					resource.TestCheckResourceAttr(fqrn, "rules.0.criteria.0.multi_license_permissive", "false"),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckLicensePolicyDestroy(id string) func(*terraform.State) error {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[id]
+		if !ok {
+			return fmt.Errorf("error: Resource id [%s] not found", id)
+		}
+		provider, _ := testAccProviders["xray"]()
+		_, resp, _ := getPolicy(rs.Primary.ID, provider.Meta().(*resty.Client))
+
+		if resp.StatusCode() == http.StatusOK {
+			return fmt.Errorf("error: Policy %s still exists", rs.Primary.ID)
+		}
+		return nil
+	}
+}
+
+func testAccXrayLicensePolicy_badLicense(resourceName, name, description, ruleName string, rangeTo int) string {
```

Comment:
You should move this to using a template

Reply:
done